gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
as you pointed out, require relative is commonly used in the test context. I don't think we'd be the first.

Earlgrey isn't defined in the lib folder & doesn't import the related files then that's a bug the tests should catch. I wouldn't want the require call to succeed if that file has been moved or the contents were deleted.